### PR TITLE
minor edits in Rerendering feedstocks

### PR DESCRIPTION
**Pull request by prachi237**
PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

**Comment by prachi237** [comment]
@viniciusdc   sir review the docs please

File: src/maintainer/updating_pkgs.rst [hunk 1]
```diff
@@ -103,32 +103,32 @@ Here we assume that you would like to update the feedstock ``<feedstock>``. Feed
...
 ================
 
-Please follow the following guidelines when updating recipes.
+Please follow the following guidelines while updating recipes:
 
-1. Always use a fork of the feedstock when updating the recipe.
+1. Always use a fork of the feedstock while updating the recipe.
 2. When a package's version is not changed, but other metadata or parts of the recipe are changed, increase the build number by ``1``.
-3. When shipping a new version of your package, reset the build number to ``0``.
+3. When you ship a new version of your package, reset the build number to ``0``.
```
Comment (isuruf) [inline src/maintainer/updating_pkgs.rst:110]: `shipping` is correct here

File: src/maintainer/updating_pkgs.rst [hunk 2]
```diff
@@ -103,32 +103,32 @@ Here we assume that you would like to update the feedstock ``<feedstock>``. Feed
...
 .. _dev_update_rerender:
 
 Rerendering feedstocks
 ======================
 
-Rerendering is conda-forge's way to update the files common to all feedstocks (e.g. README, :term:`CI` configuration, pinned dependencies)
+Rerendering is conda-forge's way to update the files common to all feedstocks (e.g. README, :term:`CI` configuration, pinned dependencies).
 
-Rerendering can be done in two ways:
+Rerendering can be done in two ways, that are mentioned below:
```
Comment (isuruf) [inline src/maintainer/updating_pkgs.rst:120]: No need to have more words.

File: src/maintainer/updating_pkgs.rst [hunk 3]
```diff
@@ -103,32 +103,32 @@ Here we assume that you would like to update the feedstock ``<feedstock>``. Feed
...
 
- #. Using the webservice to run conda-smithy on the cloud by adding the comment ``@conda-forge-admin please rerender`` (see :ref:`dev_admservice`).
+ #. Using the webservice to run conda-smithy on the cloud by adding the comment ``@conda-forge-admin please rerender`` (Visit :ref:`dev_admservice`).
```
Comment (isuruf) [inline src/maintainer/updating_pkgs.rst:122]: `see` is the usual term used when referencing other parts of docs

File: src/maintainer/updating_pkgs.rst [hunk 4]
```diff
@@ -148,13 +148,13 @@ To do this drop ``-c auto`` from the command.
...
 
-We need to re-render when there are changes to the following parts of the feedstock:
+We need to re-render when there are changes in the following parts of the feedstock:
 
-- the platform configuration (``skip`` sections);
-- the ``yum_requirements.txt``;
-- updates in the build matrix due to new versions of Python, NumPy, PERL, R, etc.
-- updates in conda-forge pinning that affect the feedstock
-- build issues that a feedstock configuration update will fix (follow us on `gitter <https://gitter.im/conda-forge/conda-forge.github.io>`_ to know about those);
+- The platform configuration (``skip`` sections).
```
Comment (isuruf) [inline src/maintainer/updating_pkgs.rst:153]:
```suggestion
- Platform configuration (``skip`` sections).
```

File: src/maintainer/updating_pkgs.rst [hunk 5]
```diff
@@ -148,13 +148,13 @@ To do this drop ``-c auto`` from the command.
...
+- The ``yum_requirements.txt``.
```
Comment (isuruf) [inline src/maintainer/updating_pkgs.rst:154]:
```suggestion
-  ``yum_requirements.txt`` or ``conda-forge.yml``.
```

Comment (prachi237) [reply src/maintainer/updating_pkgs.rst:154]: Thanku @isuruf  sir for ur feedback.

**Pull request merged**